lib/core/src/Cardano/Wallet.hs

```diff
@@ -681,7 +683,9 @@ newWalletLayer tracer bp db nw tl = do
         -- the wallet disappear within these calls, so either the wallet is
         -- there and they all succeed, or it's not and they all fail.
         DB.withLock db $ do
-            (cp, meta) <- _readWallet wid
+            (wallet, meta) <- _readWallet wid
+            let cps = applyBlocks @s @t blocks wallet
```

Comment:
I would like to suggest using `txs_cps` here, as it conveys the fact that we have a list of **pairs** of transactions (`txs`) and checkpoints (`cps`), rather than just a list of checkpoints.

Reply:
This `_` really troubled me to be honest :sweat_smile: ... Plus, txs are somewhat part of the "checkpoint" concept. I do agree however that we could better convey the underlying data structure with a more adequate name. `txsCp` will do I suppose?

Reply:
The issue I have with something like `txsCp` is that it seems to imply a singular item, when in fact we a list of things.

This goes against the common convention for using the `s` suffix to denote a plural. So for example, `ks` for a list of keys, and `vs` for a list of values, `kvs` for a list of key-value pairs, and so on.

If we have to use camelCase, then perhaps `txsCps`?

Reply:
I went for: 

```hs
            let (txs, cps) = NE.unzip $ applyBlocks @s @t blocks wallet
```

Since in the end, most subsequent calls where doing stuff like `fst <$> ...` or `snd <$> ...`